pandas/tests/frame/methods/test_drop_duplicates.py

```diff
@@ -471,3 +471,21 @@ def test_drop_duplicates_non_boolean_ignore_index(arg):
     msg = '^For argument "ignore_index" expected type bool, received type .*.$'
     with pytest.raises(ValueError, match=msg):
         df.drop_duplicates(ignore_index=arg)
+
+
+def test_drop_duplicates_pos_args_deprecation():
+    # GH#41485
+    df = DataFrame({"a": [1, 1, 2], "b": [1, 1, 3], "c": [1, 1, 3]})
+
+    msg = (
+        "In a future version of pandas all arguments of "
+        "DataFrame.drop_duplicates except for the argument 'subset' "
+        "will be keyword-only"
+    )
+
+    with tm.assert_produces_warning(FutureWarning, match=msg):
+        result = df.drop_duplicates(["b", "c"], "last")
+
+    expected = DataFrame({"a": [1, 2], "b": [1, 3], "c": [1, 3]}, index=[1, 2])
+
+    tm.assert_frame_equal(expected, result)
```

Comment:
I think we can remove all the extra newlines here, this test could read as a single paragraph

Reply:
I made this change to all my tests in fbf70a2, they now all read as one paragraph.